src/Parsing/ParserState.php

```diff
@@ -424,21 +424,21 @@ public function strlen($sString): int
     }
 
     /**
-     * @param int $iStart
+     * @param int $start
      * @param int $length
      */
-    private function substr($iStart, $length): string
+    private function substr($start, $length): string
```

Comment:
The [equivalant PHP function](https://www.php.net/manual/en/function.substr.php) has this parameter as `$offset`, which I think is better.